### DISCUSSION TITLE
Tech Stacks

### DISCUSSION THREAD

**Discussion by OmChow** [ummdotco/umm#15] [Q&A] [answered]
### What are the tech stacks we are using?

**Comment by Supsource** [comment, marked as answer]
Many thanks for asking!

**JavaScript** serves as our main programming language, while **MongoDB** serves as our document database. For our web development requirements, we use the **Express.js** framework, which is built on top of **Node.js**. **React Native.js**, a well-liked JavaScript framework for creating cross-platform mobile applications, is what we're employing on the client side. And last, our main web server in **Node.js**.

**Comment by OmChow** [comment]
Cool! I wanna contribute to Umm front-ebd. I really love what you are
building, and I'm really passionate about this space.
